parser/tavor.go

```diff
@@ -1525,11 +1525,26 @@ func (p *tavorParser) parseTypedTokenDefinition(variableScope *token.VariableSco
 			}
 
 			c = p.scan.Scan()
+
+			// optional sign (+/-)
+			neg := false
```

Comment:
and add the prefix here. There is no need for the "neg" variable. Just the "prefix" variable is fine.